internal/service/autoscaling/policy.go

```diff
@@ -761,6 +1122,15 @@ func flattenPredictiveScalingMetricSpecifications(metricSpecification []*autosca
 	if metricSpecification[0].PredefinedScalingMetricSpecification != nil {
 		metricSpecificationFlat["predefined_scaling_metric_specification"] = flattenPredefinedScalingMetricSpecification(metricSpecification[0].PredefinedScalingMetricSpecification)
 	}
+	if metricSpecification[0].CustomizedScalingMetricSpecification != nil {
+		metricSpecificationFlat["customized_scaling_metric_specification"] = flattenCustomizedScalingMetricSpecification(metricSpecification[0].CustomizedScalingMetricSpecification)
+	}
+	if metricSpecification[0].CustomizedLoadMetricSpecification != nil {
+		metricSpecificationFlat["customized_load_metric_specification"] = flattenCustomizedLoadMetricSpecification(metricSpecification[0].CustomizedLoadMetricSpecification)
+	}
+	if metricSpecification[0].CustomizedCapacityMetricSpecification != nil {
+		metricSpecificationFlat["customized_capacity_metric_specification"] = flattenCustomizedCapacityMetricSpecification(metricSpecification[0].CustomizedCapacityMetricSpecification)
+	}
```

Comment:
Small nit: Sort these alphabetically as well. Same for the flatten functions themselves below in the same file